src/Form/Type/MoneyPHPType.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Money\MoneyBundle\Form\Type;
+
+use Money\Money;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class MoneyPHPType extends AbstractType
```

Comment:
Hi sorry for the late review.

Do we need to be verbose here? Maybe `MoneyType` would be better... WDYT?

Reply:
Hello @gmponos, Symfony has its own money type so we have 2 choices — I think:

Reply:
Hello @gmponos, Symfony has its own money type so we have 2 choices — I think — here:
1. use a custom block name

Reply:
Hello @gmponos, since Symfony has its own money type we have 2 choices — I think — here:
- use a custom block name
- use a more explicit class name (generating its own block name)

I went for the 2nd option to provide a better developer experience